common/script_values/wc_men_at_arms_values.txt

```diff
@@ -81,6 +83,22 @@ culture_ai_weight_caster = {
 		}
 		add = -60
 	}
+	if = { # Cultures with a special MaA of that type should avoid the base version of the unit
```

Comment:
Errors:
```
[00:39:09][pdx_persistent_reader.cpp:421]: unknown variable [high_maint_mult].  file: common/script_values/00_men_at_arms_values.txt line: 14
[00:39:09][pdx_persistent_reader.cpp:438]: Failed to parse advanced statement at  file: common/script_values/00_men_at_arms_values.txt line: 14
[00:39:09][pdx_persistent_reader.cpp:421]: unknown variable [high_maint_mult].  file: common/script_values/00_men_at_arms_values.txt line: 18
[00:39:09][pdx_persistent_reader.cpp:438]: Failed to parse advanced statement at  file: common/script_values/00_men_at_arms_values.txt line: 18
[00:39:09][pdx_persistent_reader.cpp:421]: unknown variable [high_maint_mult].  file: common/script_values/00_men_at_arms_values.txt line: 22
[00:39:09][pdx_persistent_reader.cpp:438]: Failed to parse advanced statement at  file: common/script_values/00_men_at_arms_values.txt line: 22
[00:39:09][pdx_persistent_reader.cpp:421]: unknown variable [high_maint_mult].  file: common/script_values/00_men_at_arms_values.txt line: 26
[00:39:09][pdx_persistent_reader.cpp:438]: Failed to parse advanced statement at  file: common/script_values/00_men_at_arms_values.txt line: 26
[00:39:09][pdx_persistent_reader.cpp:421]: unknown variable [high_maint_mult].  file: common/script_values/00_men_at_arms_values.txt line: 30
[00:39:09][pdx_persistent_reader.cpp:438]: Failed to parse advanced statement at  file: common/script_values/00_men_at_arms_values.txt line: 30
[00:39:09][pdx_persistent_reader.cpp:421]: unknown variable [high_maint_mult].  file: common/script_values/00_men_at_arms_values.txt line: 34
[00:39:09][pdx_persistent_reader.cpp:438]: Failed to parse advanced statement at  file: common/script_values/00_men_at_arms_values.txt line: 34
[00:39:09][pdx_persistent_reader.cpp:421]: unknown variable [high_maint_mult].  file: common/script_values/00_men_at_arms_values.txt line: 38
[00:39:09][pdx_persistent_reader.cpp:438]: Failed to parse advanced statement at  file: common/script_values/00_men_at_arms_values.txt line: 38
```

Reply:
I asked about this in a comment, I have checked this a few times now and I think ercarp took a peek too. I dont see why this issue is popping up at all. The format is straight out of vanilla.

Reply:
> I asked about this in a comment, I have checked this a few times now and I think ercarp took a peek too. I dont see why this issue is popping up at all. The format is straight out of vanilla.

Fix brackets first.

Reply:
Fixed brackets, this problem seems unrelated

Reply:
Fixed in 2e1a8b1dcb185ee957c15ea6f15ec3c3cfb0d3cf.